Form/Type/DateRangeType.php

```diff
@@ -20,15 +20,28 @@
 class DateRangeType extends AbstractType
 {
     /**
-     * @var TranslatorInterface
+     * @var TranslatorInterface|null
+     *
+     * @deprecated translator property is deprecated since version 3.x, to be removed in 4.0
```

Comment:
This PR is duplicate of my PR #334, if the `3.x` vs `3.1` isn't problem, why closing the original one?